src/components/editor/property/InputLiteral.jsx

```diff
@@ -96,24 +96,31 @@ export class InputLiteral extends Component {
       const itemId = obj.id || shortid.generate()
 
       return <div id="userInput" key = {itemId} >
-        {obj.content}
-        <button
-          id="deleteItem"
-          type="button"
-          onClick={this.handleDeleteClick}
-          key={`delete${obj.id}`}
-          data-item={itemId}
-          data-label={this.props.formData.uri}
-        >X
-        </button>
+        <div
+          className="rbt-token rbt-token-removeable">
+          {obj.content}
+          <button
+            id={`delete${obj.id}`}
+            type="button"
+            onClick={this.handleDeleteClick}
+            key={`delete${obj.id}`}
```

Comment:
is `obj.id` enough of a key here?

Reply:
Yes-ish. The key for the delete button is prefixed with _delete_; the key for the edit button is prefixed with _edit_. So it could go either way, but this is consistent with the edit button.